plugins/inventory/cobbler.py

```diff
@@ -19,7 +19,7 @@
       plugin:
         description: The name of this plugin, it should always be set to C(cobbler) for this plugin to recognize it as it's own.
```

Comment:
```suggestion
        description: The name of this plugin, it should always be set to C(cobbler) for this plugin to recognize it as it's own.
```
shouldn't it be changed too? and in the other files here